news/feature-4209.md

```diff
@@ -0,0 +1,43 @@
+`network`/`syslog`/`tls` context options: SSL_CONF_cmd support
+
+SSL_CONF_cmd TLS configuration support for `network()` and `syslog()` driver has been added.
```

Comment:
I think the description here is a bit too technical and tied into OpenSSL function calls. It would make sense to explain that openssl offers an alternative, software-independent configuration mechanism through the SSL_CONF_cmd interface and this implements that. We could also mention that in apache mod_ssl a similar mechanism exists.